apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -290,6 +299,20 @@ export const FileBrowser = React.memo(() => {
     [newFolder, project.folders]
   );
 
+  const downloadFile: FilesComponentProps['downloadFile'] = useMemo(
+    () => fileId => {
+      const file = project.files[fileId];
+      // NOTE - this is arbitrary and maybe we need more? Also, maybe this should be elsewhere or more easy to configure?
+      const downloadableFileType = new Set(['txt', 'html', 'css', 'py', 'csv']);
+      if (downloadableFileType.has(file.language)) {
+        fileDownload(file.contents, file.name);
+      } else {
+        alert(codebridgeI18n.downloadFileError({extension: file.language}));
```

Comment:
can you file a task to replace all the `alerts` in this file with our own dialogs?

Reply:
I went one better and removed the dialog entirely. 😆 

Upon reflection, it was silly to have the `download file` menu item always appear and pop up an alert if the file couldn't be downloaded, so I retooled it to only display the `download file` option if the file's type is one of the one's in the config's `editableFileTypes`.